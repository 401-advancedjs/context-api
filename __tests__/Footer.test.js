import React from 'react';
import Footer from '../component/Footer/Footer.js';

import renderer from 'react-test-renderer';

describe('<Footer />', () => {
  test('basic rendering', () => {
    const mountedFooter = shallow(<Footer />);
    expect(mountedFooter.find('Footer')).toBeTruthy();
  });

  test('rendering follows the snapshot', () => {
    const snapshot = renderer.create(<Footer />).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});