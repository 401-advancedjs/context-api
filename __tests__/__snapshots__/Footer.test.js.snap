// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> rendering follows the snapshot 1`] = `
<p>
  Â©2019 Code Fellows
</p>
`;
